src/util/get-all-keys.js

```diff
@@ -0,0 +1,7 @@
+
+export default function keys(obj = {}, { enumerable = false } = {}) {
+  const listOfKeys = Object[enumerable ? 'getOwnPropertyNames' : 'keys'](obj);
```

Comment:
@treshugart @nadiam84 `getOwnPropertyNames` give you all props enum and non-enum. `keys` only give you enum. What do you guys think to just rename the boolean flag to `enumOnly` or `onlyEnumerable` (help with the name) and flip the return values (`enumerable ? 'keys' : 'getOwnPropertyNames'`) ?